stylesheets/index.css

```diff
@@ -0,0 +1,27 @@
+
+#intro{
+    margin: 50px;
+    margin-left: 150px;
+    margin-right: 150px;
```

Comment:
This can be simplified to `margin: 50px 150px;` The first number sets the top and bottom, the second number sets left and right. See this for more information: https://developer.mozilla.org/en-US/docs/Web/CSS/margin